src/main/kotlin/com/autonomousapps/internal/utils/gradleStrings.kt

```diff
@@ -46,8 +46,9 @@ private fun ResolvedDependencyResult.compositeRequest(): IncludedBuildCoordinate
   val resolved = ProjectCoordinates(
     identifier = (selected.id as ProjectComponentIdentifier).identityPath(),
     gradleVariantIdentification = gradleVariantIdentification,
-    // FIXME use 'buildState.buildIdentifier.buildPath' with Gradle 8.2+
-    buildPath = (selected.id as ProjectComponentIdentifier).build.name
+    buildPath = (selected.id as ProjectComponentIdentifier).build.let {
+      if (GradleVersions.isAtLeastGradle82) it.buildPath else @Suppress("DEPRECATION") it.name
```

Comment:
Are we ok with the fact that Gradle 8.2+ uses `:second-build`, and otherwise it's `second-build` (no `:` prefix)?

Reply:
Yes. It is fine as long as we consistently use the same API in one Gradle version. We do not care much about the concrete values, but only about comparing the values to figure out if a (sub)project belongs to a certain build – can be depended on via `project(...)` – or not.